front/app/api/survey_results/types.ts

```diff
@@ -13,20 +13,31 @@ export type IParameters = {
 };
 
 export interface Answer {
-  answer: Multiloc;
-  responses: number;
+  answer: string | null;
```

Comment:
```suggestion
  answer: string | number | null;
```

I guess for linear scale answers these can be numbers too?